packages/ui5-library-generator/README.md

```diff
@@ -0,0 +1,18 @@
+# @sap-ux/generator-ui5-library
+
```

Comment:
Can we please add more info about what this generator actually does and maybe an example.

Reply:
Done